js/blocks/controls/content.js

```diff
@@ -0,0 +1,48 @@
+import FetchInput from './fetch-input';
```

Comment:
The name of this file, `content.js`, probably isn't the best. 

It's a function that content controls use: Post and Taxonomy.